### PR TITLE
UI tests TUA-[949, 308, 765, 971, 955]

### PR DESCRIPTION
**Pull request by hohashvili**
TUA 949
[Footer] Verify that when the user hovers over the 'Допомога проєкту' button, it lights up
TUA 308
[Header] Verify that user is redirected to 'Челендж' page when clicking on it and its label is underlined
TUA 765
[API. Club] Description field can't contain less than 40 symbols
TUA 971
Verify that the icon of the main category is set by default if logo is not uploaded (for a club that is in the center)
TUA 955
Verify that user can edit ‘Контакти’ fields (for a club that is in the center) with valid data, and save changes on the ‘Контакти’ tab of the ‘Редагувати гурток’ pop-up window

### CODE REVIEW

File: src/test/java/com/academy/ui/addClub/AddClubPopUpWithAdminTest.java [hunk 1]
```diff
@@ -1,11 +1,6 @@
 package com.academy.ui.addClub;
 
-import com.academy.ui.components.AddClubPopUpComponent.AddClubInputElement;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpComponent;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpSider;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpStepOne;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpStepThree;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpStepTwo;
+import com.academy.ui.components.AddClubPopUpComponent.*;
```
Comment (s-hrynus) [inline src/test/java/com/academy/ui/addClub/AddClubPopUpWithAdminTest.java:3]: fix auto importing

File: src/test/java/com/academy/ui/profilePage/EditClubCardWithManagerTest.java [hunk 2]
```diff
@@ -1,17 +1,19 @@
 package com.academy.ui.profilePage;
 
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpComponent;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpStepOne;
-import com.academy.ui.components.AddClubPopUpComponent.AddClubPopUpStepThree;
+import com.academy.ui.components.AddClubPopUpComponent.*;
```
Comment (s-hrynus) [inline src/test/java/com/academy/ui/profilePage/EditClubCardWithManagerTest.java]: same

**Review by s-hrynus — APPROVED** [review]
Good job 👍

File: src/test/java/com/academy/ui/footer/FooterComponentTest.java [hunk 3]
```diff
@@ -131,6 +133,30 @@ public void testInstagramIconClickOpensCorrespondingPage() {
...
+        WebElement footerButton = footerComponent.getDonateButton();
+        System.out.println(footerButton.getCssValue("background-color"));
+        String expectedColor = "rgba(255, 169, 64, 1)";
+        Actions actions = new Actions(driver);
+        actions.moveToElement(footerButton).perform();
+
+        try {
+            Thread.sleep(1000);
+        } catch(InterruptedException e) {
+        }
```
Comment (lhalam) [inline src/test/java/com/academy/ui/footer/FooterComponentTest.java]: ???

File: src/test/java/com/academy/ui/footer/FooterComponentTest.java [hunk 4]
```diff
@@ -131,6 +133,30 @@ public void testInstagramIconClickOpensCorrespondingPage() {
...
+    public void testDonateButtonLightsUp() {
+        homePage.scrollToFooter();
+        FooterComponent footer = homePage.getFooter();
+        softAssert.assertTrue(footer.isDonateBlockIsDisplayed());
+
+        WebElement footerButton = footerComponent.getDonateButton();
+        System.out.println(footerButton.getCssValue("background-color"));
+        String expectedColor = "rgba(255, 169, 64, 1)";
+        Actions actions = new Actions(driver);
+        actions.moveToElement(footerButton).perform();
```
Comment (lhalam) [inline src/test/java/com/academy/ui/footer/FooterComponentTest.java]: move PO

**Pull request merged**